src/components/ValidatedInput/ValidFieldPassword.tsx

```diff
@@ -27,6 +27,9 @@ export interface ValidFieldPasswordProps extends Omit<ValidFieldTextProps, "type
   customPattern?: string;
   hasPasswordStrengthMeter?: boolean;
   zxcvbnOption?: OptionsType;
+  dataQaPasswordStrength?: string;
```

Comment:
could we make it as one object so that we won't have to add plenty of dataQa props now and in the future (if we need to)? what i mean is like
type DataQa: string | null
interface StrengthMeterDataQa {
  bar:? DataQa
  barProps?: DataQa
  label?: DataQa
}

interface ValidFieldPasswordProps {
...
  strengthMeterDataQa?: StrengthMeterDataQa
}

wdyt?

Reply:
i think this is better approach
will implement it